http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/CommonTestData.scala

```diff
@@ -177,6 +181,7 @@ trait CommonTestData { this: ScalaFutures =>
     labels = Map(),
     jupyterExtensionUri = Option(GcsPath(GcsBucketName("bucketName"), GcsObjectName("extension"))),
     jupyterUserScriptUri = Option(GcsPath(GcsBucketName("bucketName"), GcsObjectName("userScript"))),
+    jupyterStartUserScriptUri = Option(GcsPath(GcsBucketName("bucketName"), GcsObjectName("startScript"))),
```

Comment:
minor, Use `Option(xxx)` does an additional `null` check, so here you can just use `Some` now that there's no way `GcsPath(GcsBucketName("bucketName"), GcsObjectName("startScript"))` is null

Reply:
Done